src/runtime/rpc/rpc_host_port.h

```diff
@@ -118,6 +120,22 @@ inline bool operator==(const host_port &hp1, const host_port &hp2)
 
 inline bool operator!=(const host_port &hp1, const host_port &hp2) { return !(hp1 == hp2); }
 
+inline bool operator<(const host_port &hp1, const host_port &hp2)
```

Comment:
Where would `operator<` be used ?

Reply:
thrift struct `optional map<host_port, xxx>`